src/main/webapp/WEB-INF/views/driver/addDriverToCarForm.jsp

```diff
@@ -0,0 +1,14 @@
+<%@ page contentType="text/html;charset=UTF-8" %>
+<html>
+<head>
+    <title>Add Driver</title>
+</head>
+<body>
+<h1>Fill all of the fields below</h1>
+<form method="post" action="${pageContext.request.contextPath}/cars/drivers/add">
+    Car ID <input type="text" name="car"><br>
+    Driver ID<input type="text" name="driver"><br>
```

Comment:
check names of parameters, will you be able to get them in controller?

add type=number as well